index.html

```diff
@@ -50,6 +50,185 @@ <h1>Hey there. I'm Nicolas<br><span>I'm a software developer</span></h1>
+                    
+                    <div class="work-info">
+                        <h4>Multi-Post Stories Gain+Glory</h4>
+                        <ul class="works-languajes-container">
+                            <li><p class="works-languajes">Ruby on rails</p></li>
+                            <li><p class="works-languajes">css</p></li>
+                            <li><p class="works-languajes">JavaScript</p></li>
+                            <li><p class="works-languajes">html</p></li>
+                        </ul>
+                        <a href="#" class="button">See Project</a>
+
+                    </div>
+
+                </div>
+
+                <div class="work">
+                    <div class="img-placeholder"></div>
+                    
+                    <div class="work-info">
+                        <h4>Multi-Post Stories Gain+Glory</h4>
+                        <ul class="works-languajes-container">
+                            <li><p class="works-languajes">Ruby on rails</p></li>
+                            <li><p class="works-languajes">css</p></li>
+                            <li><p class="works-languajes">JavaScript</p></li>
+                            <li><p class="works-languajes">html</p></li>
+                        </ul>
+                        <a href="#" class="button">See Project</a>
+
+                    </div>
+
+                </div>
+
+                <div class="work">
+                    <div class="img-placeholder"></div>
+                    
+                    <div class="work-info">
+                        <h4>Multi-Post Stories Gain+Glory</h4>
+                        <ul class="works-languajes-container">
+                            <li class="works-languajes">Ruby on rails</li>
+                            <li class="works-languajes">css</li>
+                            <li class="works-languajes">JavaScript</li>
+                            <li class="works-languajes">html</li>
+                        </ul>
+                        <a href="#" class="button">See Project</a>
+
+                    </div>
+
+                </div>    
+
+            </div>
+        </section>
+
+        <section id="aboutme-section">
+            <h2>About me</h2>           
+            <img src="images/Mask Group.png" alt="vector" class="vector">
+            <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+            <a href="#" class="button">Get My Resume</a>
+
+            <div class="aboutme-card">
+                <img src="images/icon-languages.png" alt="icon-languajes">
+                <h4>Languajes</h4>
+                <ul>
+                    <li>JavaScript</li>
+                    <li>Ruby</li>
+                    <li>Html</li>
+                    <li>CSS</li>
+                </ul>
+            </div>
```

Comment:
- [ ] Its awesome that you make technology in each category(e.g., "JavasScript", "HTML" and "CSS") a `ul` with each being inside `li`, But we also need to follow the same structure for technology categories ("Languages", "Frameworks" and "Skills"). Please make sure the technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`

Reply:
You mean the whole card of each category should be in a li tag and the three categories should be inside an ul tag?

Reply:
Yes @NicolasAndrehh.